disco_aws_automation/disco_elastigroup.py

```diff
@@ -34,11 +35,13 @@ def token(self):
         Environment variable example:
         SPOTINST_TOKEN=d7e6c5abb51bb04fcaa411b7b70cce414c931bf719f7db0674b296e588630515
         """
-        try:
-            token = os.environ['SPOTINST_TOKEN']
-            return token
-        except KeyError:
-            logger.info('Create environment variable "SPOTINST_TOKEN"')
+        if os.environ.get('SPOTINST_TOKEN'):
+            return os.environ.get('SPOTINST_TOKEN')
+        else:
+            if not self.token_warning_shown:
```

Comment:
The pattern in other files seems to be that we only prepend the underscore if the field has the same name as a property in the class